src/utils/pgwire/src/lib.rs

```diff
@@ -13,6 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 #![feature(lint_reasons)]
+#![feature(generators, proc_macro_hygiene, stmt_expr_attributes)]
```

Comment:
But these features make we can create a stream easily. (We can use ‘yield’, #[for_await] macro). 
Can we declare as https://github.com/taiki-e/futures-async-stream do?🤔

Reply:
Yes. But since this crate doesn't heavily depend on stream, I think it's worth sacrificing a little developer experience to be more user-friendly. Anyway, it's just a suggestion, we don't have to do it now.